tests/unit/amo/components/TestRoutes.js

```diff
@@ -15,7 +15,7 @@ describe(__filename, () => {
 
   it('renders the routes for the amo app', () => {
     const root = render();
-    expect(root.find(Route)).toHaveLength(26);
+    expect(root.find(Route)).toHaveLength(27);
```

Comment:
Hmm. I didn't know about this test. This doesn't seem useful.

Reply:
This wasn't a change request, just me thinking out loud.

Reply:
Agreed, but I wasn't sure if I should just delete it as part of this patch, so I didn't.